server/src/main/java/org/opensearch/index/store/remote/filecache/FileCacheStats.java

```diff
@@ -86,6 +90,7 @@ public void writeTo(final StreamOutput out) throws IOException {
         out.writeLong(used);
         out.writeLong(pinned);
         out.writeLong(evicted);
+        out.writeLong(removed);
```

Comment:
Does it need version specific check as well for backward compatibility during upgrades ?

@gbbafna  let me know your thoughts.